MapboxNavigation/NextBannerView.swift

```diff
@@ -0,0 +1,68 @@
+import UIKit
+
+/// :nodoc:
+@objc(MBNextInstructionLabel)
+class NextInstructionLabel: InstructionLabel { }
+
+/// :nodoc:
+@IBDesignable
+@objc(MBNextBannerView)
+class NextBannerView: UIView {
+    
+    weak var maneuverView: ManeuverView!
+    weak var instructionLabel: NextInstructionLabel!
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        commonInit()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        commonInit()
+    }
+    
+    func commonInit() {
+        setupViews()
+        setupLayout()
+    }
+    
+    func setupViews() {
+        let maneuverView = ManeuverView()
+        maneuverView.translatesAutoresizingMaskIntoConstraints = false
+        addSubview(maneuverView)
+        self.maneuverView = maneuverView
+        
+        let instructionLabel = NextInstructionLabel()
+        instructionLabel.shieldHeight = NextInstructionLabel.appearance().font.pointSize
```

Comment:
Tests are failing because font is nil here.

Reply:
@frederoni hopefully fixed with https://github.com/mapbox/mapbox-navigation-ios/pull/819/commits/d8b34885ffee4af2e86e869a28b267f8f76563af